django/db/models/sql/compiler.py

```diff
@@ -360,6 +375,27 @@ def compile(self, node, select_format=False):
             return node.output_field.select_format(self, sql, params)
         return sql, params
 
+    def get_combinator_sql(self, combinator, all):
```

Comment:
Small nit: I would suggest using a name that is not a built-in.  Perhaps `all_` or `_all`.